src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/JurisdictionToUserMapping.java

```diff
@@ -11,7 +12,7 @@
 @ConfigurationProperties(prefix = "idam")
 public class JurisdictionToUserMapping {
 
-    private Map<String, Credential> users;
+    private Map<String, Credential> users = ImmutableMap.of();
```

Comment:
You can do `Collections.emptyMap()` looks more readable imo

Reply:
Good point, we reasign this field in the setter as well, 'immutable' part doesn't make much sense here

Reply:
Actually I can't ...  it will through a not supported exception. 
I tried this first and the test failed.